mobile/lib/ui/settings/TEMP/lock_screen_option_confirm_password.dart

```diff
@@ -0,0 +1,166 @@
+import "package:flutter/material.dart";
+import "package:flutter/services.dart";
+import "package:photos/core/configuration.dart";
+import "package:photos/generated/l10n.dart";
+import "package:photos/theme/ente_theme.dart";
+import "package:photos/ui/common/dynamic_fab.dart";
+import "package:photos/ui/components/buttons/icon_button_widget.dart";
+import "package:photos/ui/components/text_input_widget.dart";
+
+class LockScreenOptionConfirmPassword extends StatefulWidget {
+  const LockScreenOptionConfirmPassword({
+    super.key,
+    required this.password,
+  });
+  final String password;
+
+  @override
+  State<LockScreenOptionConfirmPassword> createState() =>
+      _LockScreenOptionConfirmPasswordState();
+}
+
+class _LockScreenOptionConfirmPasswordState
+    extends State<LockScreenOptionConfirmPassword> {
+  final _confirmPasswordController = TextEditingController(text: null);
```

Comment:
Make sure this is disposed.

Reply:
Okay. Just mention why this isn't disposed in a comment over it, just for context in future.